packages/console/src/assets/docs/guides/m2m-general/README.mdx

```diff
@@ -54,12 +57,12 @@ In the API Resource tab, find the API identifier that the app needs to access. I
     <a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization#basic_authentication">
       Basic authentication
     </a>
-    , where username is the App ID, and password is the App Secret.
+    , where username is the App ID, and password is the App Secret. The credential is the username and password combined with a colon, encoded in base64.
   </li>
   <li>Carry the body data</li>
 </ul>
 
-```jsonc
+```json
```

Comment:
should be jsonc (stands for "josn comment")?

Reply:
Our code editor doesn't seem to support jsonc, and it was totally plain text
<img width="695" alt="image" src="https://github.com/logto-io/logto/assets/12833674/6ce4adf8-e63f-4efe-b944-8399c69ec44d">